setupdune.sh

```diff
@@ -24,8 +45,22 @@ install-dune() {
   (set -x; dune --version)
 }
 
-lock() {
-  (set -x; cd "$SETUPDUNEDIR" && dune pkg lock)
+enable_pkg() {
+  case "$(dune --version)" in
+    3.19*|3.20*)
+      (set -x; cd "$SETUPDUNEDIR" && test -d dune.lock) || dune_aux pkg lock
+      ;;
+    *)
+      CONFIG_DIR="${XDG_CONFIG_HOME:-$HOME/.config}/dune"
+      mkdir -p "$CONFIG_DIR"
```

Comment:
I would put the `mkdir -p` in the else branch. Not a big difference but it felt a bit odd reading the code to create the directory and then check for its contents afterwards.